frontend/public/extend/kubevirt/components/vm-list.jsx

```diff
@@ -0,0 +1,138 @@
+import * as React from 'react';
+import * as _ from 'lodash-es';
+import * as classNames from 'classnames';
+
+import { ListHeader, ColHead, List, ListPage, ResourceRow } from '../../../components/factory';
+import { Firehose, ResourceLink, ResourceKebab, Kebab } from '../../../components/utils';
+import { VirtualMachineModel, VirtualMachineInstanceModel, PodModel, NamespaceModel } from '../../../models';
+import { referenceForModel } from '../../../module/k8s';
+
+import { startStopVmModal } from './modals';
+import { getResourceKind, getLabelMatcher, findVMI, findPod, getFlattenForKind, getVmStatus } from '../module/k8s/vms';
+import * as s from '../strings';
+
+const VMHeader = (props) => <ListHeader>
+  <ColHead {...props} className="col-lg-2 col-md-2 col-sm-2 col-xs-6" sortField="metadata.name">Name</ColHead>
+  <ColHead {...props} className="col-lg-2 col-md-2 col-sm-2 col-xs-6" sortField="metadata.namespace">Namespace</ColHead>
+  <ColHead {...props} className="col-lg-2 col-md-2 col-sm-2 hidden-xs" sortField="spec.running">State</ColHead>
+  <ColHead {...props} className="col-lg-2 col-md-2 col-sm-2 hidden-xs" sortField="metadata.phase">Phase</ColHead>
+  <ColHead {...props} className="col-lg-2 col-md-2 col-sm-2 hidden-xs">Virtual Machine Instance</ColHead>
+  <ColHead {...props} className="col-lg-2 col-md-2 col-sm-2 hidden-xs">Pod</ColHead>
```

Comment:
You should only need to say `col-sm-2` on each of these if md and lg are the same.

Reply:
Agreed, will do.